challenge-030/izifresh/Perl 5/numbers

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env perl
+package Numbers; 
+
+use strict; 
+use warnings;
+use v5.10;
+ for my $i (0..10) {
+   for my $j (0..10) {
+     for my $k (0..10) { 
+       if (($i+$j+$k) == 12) {
+         if ( $i % 2 == 0 or $j % 2 == 0 or $k % 2 == 0 ) {
```

Comment:
DRY (Don't repeat yourself)!
`if ( $i % 2 == 0 or $j % 2 == 0 or $k % 2 == 0 ) {`
Shorter and more expressive version:
`if ( grep { $_ % 2 == 0 } $i, $j, $k ) {`
But the coolest source code is the source code that you can read as usual text, so, we can use vanilla module to become even more expressive:
```
# perldoc List::Util
use List::Util qw(any);
...
if ( any { $_ % 2 == 0 } $i, $j, $k ) {
```